TransformtoIqt - Integration Defect (#36766)
mantidproject/mantid: docs/source/release/v6.9.0/Framework/Algorithms/Bugfixes/36766.rst

> - Fixes a bug caused by division by 0 when normalizing
> - Adds a unit test for this

Reviewer: Thanks for adding this release note! We can add a link to the algorithm which had the fix:
```suggestion
- Fixes a bug in the :ref:`Integration <algm-Integration>` algorithm caused by division by 0 when normalizing.
```

No need to mention the unit test because it isn't a user-facing change
